### ISSUE TITLE
Image not found or type unknown for local images only after upgrading from 0.8.5 to 0.8.6

### ISSUE THREAD

**Issue by Arthi93** [dompdf/dompdf#2261]
Hi,

after upgrading from version 0.8.5 to version 0.8.6 my local images stopped working, showing message "Image not found or type unknown". Linking image from remote host works fine.

i use image like this:
`<img src="images/logo.jpg" alt="logo"/>`

i've also tried full path like this: 
`<img src="var/www/myproject/images/logo.jpg" alt="logo"/>`

After downgrading dompdf back to 0.8.5 image is working again.

I have php , html5 parser and remote enabled:
```
$options = new Dompdf\Options();
$options->setIsPhpEnabled(true);
$options->setIsHtml5ParserEnabled(true);
$options->setIsRemoteEnabled(true);
```

Any suggestions what might be the reason for that? Im loading dompdf via composer.

**Comment by ThomasLandauer** [comment]
Duplicate of https://github.com/dompdf/dompdf/issues/2229
